doc/api/errors.md

```diff
@@ -2476,6 +2469,17 @@ removed: v10.0.0
 
 The `repl` module was unable to parse data from the REPL history file.
 
+<a id="ERR_SERVER_NOT_RUNNING"></a>
+### `ERR_SERVER_NOT_RUNNING`
+<!-- YAML
+added: v9.0.0
+removed: REPLACEME
+-->
+
+The [`server.close()`][] method was called when a `net.Server` was not
+running. This applies to all instances of `net.Server`, including HTTP, HTTPS,
+and HTTP/2 `Server` instances.
+
```

Comment:
Why move?

Reply:
Along with this change, this error becomes a legacy error, and for this reason it is moved to a proper section of this document.

I've gotten a similar question a while ago, maybe it would make sense to separate the errors and legacy errors page into a separate document to avoid confusion during reviews.